staging/src/k8s.io/legacy-cloud-providers/azure/azure.go

```diff
@@ -161,12 +158,6 @@ type Config struct {
 	CloudProviderBackoffDuration int `json:"cloudProviderBackoffDuration,omitempty" yaml:"cloudProviderBackoffDuration,omitempty"`
 	// Backoff jitter
 	CloudProviderBackoffJitter float64 `json:"cloudProviderBackoffJitter,omitempty" yaml:"cloudProviderBackoffJitter,omitempty"`
-	// Backoff mode, options are v2 and default.
-	// * default means two-layer backoff retrying, one in the cloud provider and the other in the Azure SDK.
-	// * v2 means only backoff in the Azure SDK is used. In such mode, CloudProviderBackoffDuration and
-	//   CloudProviderBackoffJitter are omitted.
-	// "default" will be used if not specified.
-	CloudProviderBackoffMode string `json:"cloudProviderBackoffMode,omitempty" yaml:"cloudProviderBackoffMode,omitempty"`
```

Comment:
Is this a breaking change on the config or will this field just be ignored?

Reply:
this field would be ignored